meta/main.yml

```diff
@@ -41,4 +41,6 @@ galaxy_info:
     - rhel
     - ubuntu
 
-dependencies: []
+dependencies:
+  - robertdebock.bootstrap
```

Comment:
The bootstrap  role is not a "hard requirement". Systems can also be prepared by something else, as long as they have python and sudo, Ansible will be able to manage the machine.

I've added the bootstrap dependency in `molecule/*/playbook.yml` to prepare a container. Most are pretty dressed down, so do not have python.

Reply:
So add only reboot then?